jest.config.mjs

```diff
@@ -0,0 +1,28 @@
+/*
+ * For a detailed explanation regarding each configuration property, visit:
+ * https://jestjs.io/docs/configuration
+ */
+
+export default {
+   globals: {
+      __DEV__: true,
+   },
+   // Jest assumes we are testing in node environment, specify jsdom environment instead
+   testEnvironment: "node",
+   // Needed in JS codebases too because of feature flags
+   coveragePathIgnorePatterns: ["/node_modules/", ".d.ts$"],
+   testMatch: ["<rootDir>/tests/unit/**/*.spec.js"],
+   moduleFileExtensions: ["js"],
+   moduleNameMapper: {
+      "^src/(.*)$": "<rootDir>/src/$1",
+   },
+   // transform: {
+   //   ".*\\.js$": "babel-jest",
+   // },
+   transformIgnorePatterns: ["<rootDir>/node_modules/"],
+   // testResultsProcessor: "jest-sonar-reporter",
```

Comment:
and this one?

Reply:
Vincent told me that he will setup sonar for this repo and once that's done, I can uncomment this line?

Reply:
ok!